src/sage/calculus/functional.py

```diff
@@ -25,14 +25,16 @@
     sage: expand((x - a)^3)
     -a^3 + 3*a^2*x - 3*a*x^2 + x^3
 """
-
 from sage.structure.element import Expression
 
 
-def simplify(f):
+def simplify(f, algorithm="maxima", **kwds):
     r"""
     Simplify the expression `f`.
 
+    See the documentation of the :meth:`simplify` method of symbolic
```

Comment:
I doubt this will have a good link as it doesn't know what class to look at. I think you need to be more specific here...